konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/impl/step/ml/ssd/SSDToBoundingBoxRunner.java

```diff
@@ -25,15 +25,19 @@
 import ai.konduit.serving.pipeline.api.data.ValueType;
 import ai.konduit.serving.pipeline.api.step.PipelineStep;
 import ai.konduit.serving.pipeline.api.step.PipelineStepRunner;
+import io.micrometer.core.instrument.util.IOUtils;
 import lombok.AllArgsConstructor;
 
+import java.io.InputStream;
 import java.util.ArrayList;
 import java.util.List;
 
 @AllArgsConstructor
 public class SSDToBoundingBoxRunner implements PipelineStepRunner {
 
     protected final SSDToBoundingBoxStep step;
+    protected final String[] COCO_CLASSES_LABELS = new String[]{"person", "bicycle", "car", "motorcycle", "airplane", "bus", "train", "truck", "boat", "traffic light", "fire hydrant", "street sign", "stop sign", "parking meter", "bench", "bird", "cat", "dog", "horse", "sheep", "cow", "elephant", "bear", "zebra", "giraffe", "hat", "backpack", "umbrella", "shoe", "eye glasses", "handbag", "tie", "suitcase", "frisbee", "skis", "snowboard", "sports ball", "kite", "baseball bat", "baseball glove", "skateboard", "surfboard", "tennis racket", "bottle", "plate", "wine glass", "cup", "fork", "knife", "spoon", "bowl", "banana", "apple", "sandwich", "orange", "broccoli", "carrot", "hot dog", "pizza", "donut", "cake", "chair", "couch", "potted plant", "bed", "mirror", "dining table", "window", "desk", "toilet", "door", "tv", "laptop", "mouse", "remote", "keyboard", "cell phone", "microwave", "oven", "toaster", "sink", "refrigerator", "blender", "book", "clock", "vase", "scissors", "teddy bear", "hair drier", "toothbrush", "hair brush"};
```

Comment:
Two things to change here
(a) the labels should come from the configuration (i.e., something the user sets in SSDToBoundingBoxStep) - and they should be optinal. If the user doesn't provide a list of class labels, SSDToBoundingBoxStepRunner should still work, but just returning null for the bounding box labels
(b) the field should be `static final` not just final (so `public static final String[] COCO_CLASSES_LABELS` on SSDToBoundingBoxStep is better)